source/globalCommands.py

```diff
@@ -4982,10 +4982,11 @@ def script_connectToRemote(self, gesture: "inputCore.InputGesture"):
 
 	@script(
 		# Translators: Describes the command that creates a remote session, or disconnects it if one already exists.
-		"Toggle Remote connection",
+		description=pgettext("Remote Access", "Toggle Remote connection"),
```

Comment:
Can we stick to "remote" for consistency

```suggestion
		description=pgettext("remote", "Toggle Remote connection"),
```

Reply:
Given we're branding this as "Remote Access", isn't that how we should refer to it to translators? I'll go through and make sure we're consistent with whatever we land on, I just think using "remote" for translators and "Remote Access" for end users seems confusing.

Reply:
I think shorthand is probably fine and preferred for translators.
If not my preference would be to make it `remoteAccess` in camelCase. We use a mix of spacing and camelCase for these, but my preference is camelCase. I would avoid sentence case at the very least, i.e. `remote access`